README.md

```diff
@@ -80,3 +80,12 @@ bundle exec rails server  # or other commands
 
 ## Deployment
 Deployment is setup using Capistrano using standard [Stanford Digital Library Systems and Services practices](https://github.com/sul-dlss/DeveloperPlaybook/blob/master/best-practices/deployment.md#ruby-applications).
+
+## Getting Stated with Docker
+
+1. `docker-compose -f docker-compose.yml up`
+2. Exec into the running worker container: `docker exec -it aggworker bash`
+3. Inside the worker container, run: `./bin/setup`
+4. Exec into the running app conatiner: `docker exec -it aggapp bash`
+5. Inside the app container, run: `./bin/rails db:migrate` 
```

Comment:
tested this and i think it's sufficient to just do `docker compose exec aggapp ./bin/setup` instead of steps 2-5. you also don't need `container_name` set in the compose file to do this (although i can see why it might be helpful)

Reply:
Katie found that she had to run the db:migrate as well, but that may have been a result of some in development work. With a fresh clone and fresh containers, I also found that `./bin/setup` was sufficient. We'll keep an eye on it just in case, but I think you're right in that it can be kept out.